packages/airnode-node/src/constants.test.ts

```diff
@@ -8,17 +8,17 @@ describe('constants', () => {
     expect.assertions(constCount + 1);
     expect(Object.keys(constants).length).toEqual(12);
 
-    expect(constants.API_CALL_TIMEOUT).toEqual(10_000);
-    expect(constants.API_CALL_TOTAL_TIMEOUT).toEqual(10_000);
+    expect(constants.API_CALL_TIMEOUT).toEqual(60_000);
+    expect(constants.API_CALL_TOTAL_TIMEOUT).toEqual(60_000);
     expect(constants.BLOCK_COUNT_HISTORY_LIMIT).toEqual(300);
     expect(constants.BLOCK_COUNT_IGNORE_LIMIT).toEqual(20);
     expect(constants.BLOCK_MIN_CONFIRMATIONS).toEqual(0);
     expect(constants.CONVENIENCE_BATCH_SIZE).toEqual(10);
     expect(constants.DEFAULT_RETRY_DELAY_MS).toEqual(50);
     expect(constants.DEFAULT_RETRY_TIMEOUT_MS).toEqual(5_000);
-    expect(constants.EVM_PROVIDER_TIMEOUT).toEqual(22_000);
-    expect(constants.WORKER_CALL_API_TIMEOUT).toEqual(10_000);
-    expect(constants.WORKER_PROVIDER_INITIALIZATION_TIMEOUT).toEqual(17_000);
-    expect(constants.WORKER_PROVIDER_PROCESS_REQUESTS_TIMEOUT).toEqual(32_000);
+    expect(constants.EVM_PROVIDER_TIMEOUT).toEqual(60_000);
```

Comment:
I'm not sure about changing all of these values to 60 seconds... I'm worried that this is going to result in weird behaviour for non-serverless deployments.

I think there may have been some miscommunication here. I was only referring to removing the 500ms buffers

Reply:
And for `EVM_PROVIDER_TIMEOUT` specifically, was this changed to 22 seconds because of the `getLogs` call being quite heavy? 22 seconds seems _really_ long for this

Reply:
> I'm worried that this is going to result in weird behaviour for non-serverless deployments

Sorry about, I hadn't considered a local deployment(!), I'll revert those timeouts to the variants without buffers.

> And for EVM_PROVIDER_TIMEOUT specifically, was this changed to 22 seconds because of the getLogs call being quite heavy?

`getLogs` itself isn't heavy, but processing of the results of `getLogs` in `initializeProvider` is. I'll revert this change too.